src/app/iconography/icon-selection.ts

```diff
@@ -5,13 +5,13 @@
  */
 import { Component } from "@angular/core";
 import { CoreShapes } from "../../clarity-icons/shapes/core-shapes";
-import { BasicShapes } from "../../clarity-icons/shapes/basic-shapes";
+import { EssentialShapes } from "../../clarity-icons/shapes/essential-shapes";
```

Comment:
I've just put the old `basic-shapes` module back in the shapes directory so it will not break for people who are using `basic-shapes`.